zapd/models.py

```diff
@@ -18,6 +18,11 @@
 from utils import generate_key, is_email, is_mobile, is_address
 
 from flask_admin import BaseView
+from flask_admin.babel import lazy_gettext
```

Comment:
some of these imports are not used

Reply:
also the flask imports should be grouped with the already existing flask imports